README.md

```diff
@@ -11,9 +11,80 @@ Federalist is a unified interface for publishing static government websites. It
 
 ## Getting started
 
-To run the server, you'll need [Node.js](https://nodejs.org/download/). You'll also need [nvm](https://github.com/creationix/nvm).
+### Dependencies / Tooling
+Before you start, ensure you have the following installed:
+
+- [node](https://nodejs.org/en/download/package-manager/#osx)
+- [nvm](https://github.com/creationix/nvm#installation) or [n](https://github.com/tj/n#installation)
+- [yarn](https://yarnpkg.com/)
+- [Postgres](https://www.postgresql.org/)
+- [Docker](https://docs.docker.com/engine/installation/mac/)
+- [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html)
+
+
+### Running locally (development)
+
+1. Download or clone the `18F/federalist` repository from Github and `cd` to that directory.
+1. Run `nvm use` to ensure you are using the correct version of node.
+1. Run `yarn` to load modules and install Jekyll dependencies.
+1. Make a copy of `config/local.sample.js` and name it `local.js` and place it in the `config` folder.
+This will be the file that holds your S3 and SQS configurations.
+1. [Register a new OAuth application on GitHub](https://github.com/settings/applications/new). You'll want to use `http://localhost:1337/auth` as the "Authorization callback url". Once you have created the application, you'll see a `Client ID` and `Client Secret`. Add these values to `config/local.js`.
+    ```json
```

Comment:
oh shoot, sorry, I advised the wrong syntax hint, it should be `js` not `json`. apologies!